pkg/julia/manifest/parse_testcase.go

```diff
@@ -74,4 +74,19 @@ var (
 	juliaV1_9ShadowedDepDeps = []types.Dependency{
 		{ID: "ead4f63c-334e-11e9-00e6-e7f0a5f21b60", DependsOn: []string{"f41f7b98-334e-11e9-1257-49272045fb24"}},
 	}
+
+	juliaV1_0FormatLibs = []types.Library{
```

Comment:
https://go.dev/doc/effective_go#names
```suggestion
	juliaV10FormatLibs = []types.Library{
```

Reply:
done